website/static/js/share/index.js

```diff
@@ -19,13 +20,14 @@ ShareApp.ViewModel = function() {
     self.page = 0;
     self.count = 0;
     self.results = [];
-    self.query = m.prop($osf.urlParams().q || '');
+    self.queryString = m.prop($osf.urlParams().q || '');
```

Comment:
yeah, the query string stores what the user has entered, and the filters store what we want to add to it.